apps/react/src/features/studios/pages/StudiosPage/StudioPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { memo, FC } from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { Box } from '@mui/material';
+
+import { StudiosList } from '../../components/StudiosList';
+
+import { StudioSearchBar } from '../../components/StudioSearchBar';
+
+import { StudiosSort } from '../../components/StudioSort';
+
+/** Studio page component. */
+const StudiosPageComponent: FC = () => (
+	<Box
+		component='main'
+		sx={{
+			display: 'flex',
+			flexDirection: 'row',
+			width: '100%',
+			height: 'calc(100vh - var(--header-height))',
+		}}
+	>
+		<Box
+			component='section'
+			display='flex'
+			flexDirection='column'
+			sx={{
+				width: '380px',
+				border: '1px solid rgba(0 0 0 / 15%)',
```

Comment:
MUI has `divider` color defined for borders. 

```suggestion
				borderColor: 'divider',
```

or 
```suggestion
				border:  theme => `1px solid ${theme.palette.divider}`,
```